smac/facade/smac_facade.py

```diff
@@ -240,7 +240,7 @@ def __init__(self,
                     scenario=scenario, num_params=num_params,
                     success_states=[StatusType.SUCCESS, ],
                     impute_censored_data=True,
-                    impute_state=[StatusType.TIMEOUT, ],
+                    impute_state=[StatusType.TIMEOUT, StatusType.CAPPED ],
```

Comment:
I think TIMEOUT is no longer necessary in this list as we only impute TIMEOUT with runtime < cutoff. These runs are now of status CAPPED.